### PR TITLE
chore: update css/reference to remove `page` macro

### PR DESCRIPTION
**Pull request by jasonren0403**
### Description

sync css/reference/index with newest EN document

### Motivation

continue removing `{{page}}` macro calls

### Related issues and pull requests

#3893

### CODE REVIEW

File: files/zh-cn/web/css/reference/index.md [hunk 1]
```diff
@@ -5,24 +5,22 @@ slug: Web/CSS/Reference
 
 {{CSSRef}}
 
-使用此**CSS 参考**页面以浏览按[字母索引](#索引)的所有标准 [CSS](/zh-CN/docs/CSS) 属性、[伪类](/zh-CN/docs/CSS/Pseudo-classes)、[伪元素](/zh-CN/docs/CSS/Pseudo-elements)、[数据类型](/zh-CN/docs/Web/CSS/CSS_Types)、以及[@ 规则](/zh-CN/docs/CSS/At-rule)。你也可以浏览 [按类型排列的 CSS 选择器](#选择器) 列表和 [CSS 关键概念](#概念) 列表。还有一份简短的 [DOM-CSS / CSSOM 参考](#DOM-CSS_CSSOM)。
+使用此**CSS 参考**页面以浏览按[字母索引](#索引)的所有标准 [CSS](/zh-CN/docs/Web/CSS) 属性、[伪类](/zh-CN/docs/Web/CSS/Pseudo-classes)、[伪元素](/zh-CN/docs/Web/CSS/Pseudo-elements)、[数据类型](/zh-CN/docs/Web/CSS/CSS_Types)、[功能表记](/zh-CN/docs/Web/CSS/CSS_Functions)以及[@ 规则](/zh-CN/docs/Web/CSS/At-rule)。你也可以浏览[CSS 关键概念](#概念)和[按类型排列的 CSS 选择器](#选择器)列表。还有一份简短的 [DOM-CSS / CSSOM 参考](#dom-css_cssom)。
```
Comment (jasonren0403) [inline files/zh-cn/web/css/reference/index.md:8]:
```suggestion
使用此 **CSS 参考**页面以浏览按[字母索引](#索引)的所有标准 [CSS](/zh-CN/docs/Web/CSS) 属性、[伪类](/zh-CN/docs/Web/CSS/Pseudo-classes)、[伪元素](/zh-CN/docs/Web/CSS/Pseudo-elements)、[数据类型](/zh-CN/docs/Web/CSS/CSS_Types)、[功能表记](/zh-CN/docs/Web/CSS/CSS_Functions)以及[@ 规则](/zh-CN/docs/Web/CSS/At-rule)。你也可以浏览[CSS 关键概念](#概念)和[按类型排列的 CSS 选择器](#选择器)列表。还有一份简短的 [DOM-CSS / CSSOM 参考](#dom-css_cssom)。
```

File: files/zh-cn/web/css/reference/index.md [hunk 2]
```diff
@@ -31,7 +29,7 @@ properties-list ::=
     [property : value] [; properties-list]
 ```
 
-参阅后面的 [_selector_](#选择器)、[_pseudo-element_](#pelm)、[_pseudo-class_](#pcls) 列表。每个指定值的语法取决于为每个指定属性定义的数据类型。
+参阅后面的 [_selector_](#选择器)、[_pseudo-element_](#伪类)、[_pseudo-class_](#伪元素) 列表。每个指定*值*的语法取决于为每个指定*属性*定义的数据类型。
```
Comment (jasonren0403) [inline files/zh-cn/web/css/reference/index.md:32]:
```suggestion
参阅后面的 [_选择器_](#选择器)、[_伪元素_](#伪元素)、[_伪类_](#伪类) 列表。每个指定*值*的语法取决于为每个指定*属性*定义的数据类型。
```

File: files/zh-cn/web/css/reference/index.md [hunk 3]
```diff
@@ -45,62 +43,62 @@ div.menu-bar li:hover > ul {
...
 
 ### @规则语法
 
-由于 @规则间的结构变化很大，请参阅[@规则](/zh-CN/docs/Web/CSS/At-rule)以找到你所想要的特定规则语法。
+由于 @规则间的结构变化很大，请参阅 [@规则](/zh-CN/docs/Web/CSS/At-rule)以找到你所想要的特定规则语法。
 
 ## 索引
 
-> **备注：** 此索引中的属性名称不包括与 CSS 标准名称不同的 JavaScript 名称。
+> **备注：** 此索引中的属性名称**不包括**与 CSS 标准名称不同的 [JavaScript 名称](/zh-CN/docs/Web/CSS/CSS_Properties_Reference)。
```
Comment (jasonren0403) [inline files/zh-cn/web/css/reference/index.md:54]:
```suggestion
> **备注：** 此索引中的属性名称**不包括**与 CSS 标准名称不同的 [JavaScript 名称](/zh-CN/docs/Web/CSS/Reference#索引)。
```

File: files/zh-cn/web/css/reference/index.md [hunk 4]
```diff
@@ -129,7 +128,7 @@ div.menu-bar li:hover > ul {
 ### 布局
 
 - [块级格式化上下文](/zh-CN/docs/Web/Guide/CSS/Block_formatting_context)
-- [盒式模型](/zh-CN/docs/Web/CSS/box_model)
+- [盒模型](/zh-CN/docs/Web/CSS/box_model)
```
Comment (jasonren0403) [inline files/zh-cn/web/css/reference/index.md:131]:
```suggestion
- [盒模型](/zh-CN/docs/Web/CSS/CSS_Box_Model/Introduction_to_the_CSS_box_model)
```

File: files/zh-cn/web/css/reference/index.md [hunk 5]
```diff
@@ -129,7 +128,7 @@ div.menu-bar li:hover > ul {
...
 - [包含块](/zh-CN/docs/Web/CSS/All_About_The_Containing_Block)
```
Comment (jasonren0403) [inline files/zh-cn/web/css/reference/index.md:132]:
```suggestion
- [包含块](/zh-CN/docs/Web/CSS/Containing_block)
```

File: files/zh-cn/web/css/reference/index.md [hunk 6]
```diff
@@ -152,11 +151,15 @@ div.menu-bar li:hover > ul {
...
 
-- {{domxref("CSSStyleSheet.insertRule()")}}
-- {{domxref("CSSStyleSheet.deleteRule()")}}
+- {{DOMxRef("CSSStyleSheet.insertRule()")}}
+- {{DOMxRef("CSSStyleSheet.deleteRule()")}}
+
+## 参见
+
+- [Mozilla CSS 扩展](/zh-CN/docs/Web/CSS/Mozilla_Extensions)（以 `-moz` 前缀）
+- [WebKit CSS 扩展](/zh-CN/docs/Web/CSS/Webkit_Extensions)（大多数时候以 `-webkit` 前缀）
```
Comment (jasonren0403) [inline files/zh-cn/web/css/reference/index.md:160]:
```suggestion
- [WebKit CSS 扩展](/zh-CN/docs/Web/CSS/WebKit_Extensions)（大多数时候以 `-webkit` 前缀）
```

**Review by yin1999 — APPROVED** [review]

File: files/zh-cn/web/css/reference/index.md [hunk 7]
```diff
@@ -110,12 +108,13 @@ div.menu-bar li:hover > ul {
 - [@ 规则](/zh-CN/docs/Web/CSS/At-rule)
 - [层叠](/zh-CN/docs/Web/CSS/Cascade)
 - [注释](/zh-CN/docs/Web/CSS/Comments)
-- [解释器](</zh-CN/docs/Glossary/Descriptor_(CSS)>)
+- [解释器](/zh-CN/docs/Glossary/Descriptor_(CSS))
```
Comment (yin1999) [inline files/zh-cn/web/css/reference/index.md:111]: 链接中有括号，最好加上尖括号：

```suggestion
- [解释器](</zh-CN/docs/Glossary/Descriptor_(CSS)>)
```

File: files/zh-cn/web/css/reference/index.md [hunk 8]
```diff
@@ -45,62 +43,62 @@ div.menu-bar li:hover > ul {
...
 
 ### @规则语法
 
-由于 @规则间的结构变化很大，请参阅[@规则](/zh-CN/docs/Web/CSS/At-rule)以找到你所想要的特定规则语法。
+由于 @规则间的结构变化很大，请参阅 [@规则](/zh-CN/docs/Web/CSS/At-rule)以找到你所想要的特定规则语法。
 
 ## 索引
 
-> **备注：** 此索引中的属性名称不包括与 CSS 标准名称不同的 JavaScript 名称。
+> **备注：** 此索引中的属性名称**不包括**与 CSS 标准名称不同的 [JavaScript 名称](/zh-CN/docs/Web/CSS/Reference#索引)。
```
Comment (yin1999) [inline files/zh-cn/web/css/reference/index.md:54]:
```suggestion
> **备注：** 此索引中的属性名称**不包括**与 CSS 标准名称不同的 [JavaScript 名称](#索引)。
```

File: files/zh-cn/web/css/reference/index.md [hunk 9]
```diff
@@ -31,7 +29,7 @@ properties-list ::=
     [property : value] [; properties-list]
 ```
 
-参阅后面的 [_selector_](#选择器)、[_pseudo-element_](#pelm)、[_pseudo-class_](#pcls) 列表。每个指定值的语法取决于为每个指定属性定义的数据类型。
+参阅后面的 [_选择器_](#选择器)、[_伪元素_](#伪元素)、[_伪类_](#伪类) 列表。每个指定*值*的语法取决于为每个指定*属性*定义的数据类型。
```
Comment (yin1999) [inline files/zh-cn/web/css/reference/index.md:32]:
```suggestion
参阅后面的[_选择器_](#选择器)、[_伪元素_](#伪元素)、[_伪类_](#伪类)列表。每个指定*值*的语法取决于为每个指定*属性*定义的数据类型。
```

File: files/zh-cn/web/css/reference/index.md [hunk 10]
```diff
@@ -45,62 +43,62 @@ div.menu-bar li:hover > ul {
...
+以下是一些[选择器](/zh-CN/docs/Web/CSS/CSS_Selectors)的列表，它们允许根据 DOM 内元素的各种特征对样式进行条件限制。
 
-### 分组选择器
-
-- [选择器列表](/zh-CN/docs/Web/CSS/Selector_list) `A, B`
-  - : 指定同时选择`A` 和 `B`元素。这是一种选择多个匹配元素的分组方法。
+### 基本选择器
 
-### [组合选择器](/zh-CN/docs/Web/CSS/CSS_Selectors#Combinators)
+[基本选择器](/zh-CN/docs/Web/CSS/CSS_Selectors#Simple_selectors)是选择器中最初级的，通常由它们的组合构成其他更复杂的选择器。
```
Comment (yin1999) [inline files/zh-cn/web/css/reference/index.md:64]:
```suggestion
[基本选择器](/zh-CN/docs/Web/CSS/CSS_Selectors#基本选择器)是选择器中最初级的，通常由它们的组合构成其他更复杂的选择器。
```

**Pull request merged**